tools/profile.py

```diff
@@ -216,6 +225,18 @@ def stack_id_err(stack_id):
 }
 """
 
+# pid-namespace translation
+try:
+    devinfo = os.stat("/proc/self/ns/pid")
+    bpf_text = bpf_text.replace('USE_PIDNS', "1")
+    bpf_text = bpf_text.replace('PIDNS_DEV', str(devinfo.st_dev))
+    bpf_text = bpf_text.replace('PIDNS_INO', str(devinfo.st_ino))
+    pidns_translation = "1"
```

Comment:
`pidns_translation` is unused.

Reply:
Ah right sorry, it's leftover from a previous iteration of the patch, I dropped it now.